kalite/coachreports/static/css/coachreports/report_view.css

```diff
@@ -0,0 +1,153 @@
+.selection {
```

Comment:
This file is almost completely just a [clone](http://en.wikipedia.org/wiki/Don%27t_repeat_yourself) of tabular_view.css